### ISSUE TITLE
[select] "role" of select

### ISSUE THREAD

**Issue by mahi160** [mui/base-ui#4754]
## Ask a question

I see the role of select is always combobox. Shouldn't it be listbox when single select? According to MDN, this should be the case.

https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/select#technical_summary

**Comment by atomiks** [comment]
`<Select.Trigger>` is `role="combobox"` while `<Select.List>` is `role="listbox"`. The trigger is the combobox, not a listbox
